apps/server/src/shared/domain/service/permission.service.ts

```diff
@@ -44,6 +45,9 @@ export class PermissionService {
 		return permissions;
 	}
 
+	/**
```

Comment:
Do we have a ticket to remove this service?

Reply:
yes